src/components/Chat/Chat.jsx

```diff
@@ -2,15 +2,36 @@ import React from 'react';
 import { sendChat } from './ChatActions';
 import { connect } from 'react-redux';
 import styles from './Chat.module.css';
+import classNames from 'classnames/bind';
+
+const cx = classNames.bind(styles);
 
 export class Chat extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            currentValue: ''
+            currentValue: '',
+            showChat: true
         }
         this.handleChange = this.handleChange.bind(this);
         this.sendMessage = this.sendMessage.bind(this);
+        this.showChat = this.showChat.bind(this);
+        this.updateWindowDimensions = this.updateWindowDimensions.bind(this);
+    }
+      
+    componentDidMount() {
+        this.updateWindowDimensions();
+        window.addEventListener('resize', this.updateWindowDimensions);
+    }
+      
+    componentWillUnmount() {
+        window.removeEventListener('resize', this.updateWindowDimensions);
+    }
+      
+    updateWindowDimensions() {
+        this.setState({
+            showChat: window.innerWidth > 950
```

Comment:
I'm not that keen on this way of doing it, as it then hides the chat everytime you change the screen. Can't you use css media queries?